src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/MultiplesHelper.java

```diff
@@ -324,4 +330,50 @@ public static void populateDynamicListOfficesMultiple(MultipleData multipleData,
 
     }
 
+    public static SortedMap<String, SortedMap<String, Object>> createCollectionOrderedByCaseRef(List<?> list) {
```

Comment:
Is there a more meaningful name for this method? I wouldn't know why I would use it without looking at the method code

Reply:
It's a method that essentially sorts a list of given objects which has a ethos case reference in ascending order by their referenced year then by their reference number. I thought about the name for this method for literal hours, and changed my mind like 5 times, any suggestions would be welcome.